src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -79,6 +85,67 @@ def invoke_package_manager(self, command):
             self.set_package_manager_setting(Constants.PACKAGE_MGR_SETTING_REPEAT_PATCH_OPERATION, True)
         return out
 
+    """
+        When given a string containing a pid, returns a multi-line string with a list of parent/child processes for that pid.
+        See pattern matching inside the function for details on specific strings that are accepted.
+        
+        Example:
+            input: 
+                message (string): Output from package manager: | System management is locked by the application with pid 7914 (/usr/bin/zypper).
+                
+            returns (string):
+                  PID CMD
+                 7736 /bin/bash
+                 7912  \_ python3 package_test.py
+                 7913  |   \_ sudo LANG=en_US.UTF8 zypper --non-interactive update --dry-run bind-utils
+                 7914  |       \_ zypper --non-interactive update --dry-run bind-utils
+                 7982  |           \_ /usr/bin/python3 /usr/lib/zypp/plugins/urlresolver/susecloud
+                 7984  |               \_ /usr/bin/python3 /usr/bin/azuremetadata --api latest --subscriptionId --billingTag --attestedData --signature
+                 7986  \_ python3 package_test.py
+                 8298      \_ sudo LANG=en_US.UTF8 zypper --non-interactive update --dry-run grub2-i386-pc
+    """
+    def get_process_tree_from_package_manager_msg(self, message):
+        """
+            First find pid xxxxx within output string.
+
```

Comment:
remove line break

Reply:
By line break, I meant, remove the extra blank line. Remove blank line on line 91. Try to only have blank lines between different logic parts in code, and avoid additional blank lines in other places

Reply:
Got it, thanks!